packages/event-logger/src/agent/EventLogger.ts

```diff
@@ -29,7 +33,7 @@ export class EventLogger implements IAgentPlugin {
   public async onEvent(event: LoggingEvent, context: RequiredContext): Promise<void> {
     switch (event.type) {
       case LoggingEventType.AUDIT:
-        await this.loggerLogAuditEvent({ event: event.data }, context)
+        await context.agent.loggerLogAuditEvent({ event: event.data })
```

Comment:
Probably wise to add a comment why it is calling the agent instead of this (to make sure it uses the rest client if configured)

Reply:
added a comment